src/status_im/contexts/wallet/account/bridge_to/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns status-im.contexts.wallet.account.bridge-to.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.account.bridge-to.style :as style]
+    [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
+    [status-im.contexts.wallet.common.utils :as utils]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- bridge-token-component
+  []
+  (fn [{:keys [chain-id network-name]} token]
+    (let [network           (rf/sub [:wallet/network-details-by-chain-id chain-id])
+          currency          (rf/sub [:profile/currency])
+          currency-symbol   (rf/sub [:profile/currency-symbol])
+          all-balances      (:balances-per-chain token)
+          balance-for-chain (utils/get-balance-for-chain all-balances chain-id)
+          crypto-formatted  (or (:balance balance-for-chain) "0.00")
+          fiat-value        (utils/token-fiat-value currency
+                                                    (or (:balance balance-for-chain) 0)
+                                                    token)
+          fiat-formatted    (utils/get-standard-fiat-format crypto-formatted currency-symbol fiat-value)]
+      [quo/network-list
+       {:label         (name network-name)
+        :network-image (quo.resources/get-network (:network-name network))
+        :token-value   (str crypto-formatted " " (:symbol token))
+        :fiat-value    fiat-formatted}])))
```

Comment:
@J-Son89 do you know if we have a map to get network names for the user?

Also, I think we have a helper funciton taking a crypto value and returns the string formatted :thinking:

Reply:
I think what @Pau1fitz done here looks good. (str crypto-formatted " " (:symbol token)) - this could be an i18n label with a parameter for the amount too 👍